modules/rostests/apitests/com/com_apitest.c

```diff
@@ -411,6 +411,11 @@ static KNOWN_INTERFACE KnownInterfaces[] =
 
     { ID_NAME(CLSID_ShellDesktop),              TRUE },
 
+    { ID_NAME(IID_IQueryContinue)                    },
+    { ID_NAME(IID_IUserNotification)                 },
+//  { ID_NAME(IID_IUserNotificationCallback)         }, // On Vista+
```

Comment:
Commenting these out makes no sense.

Reply:
We don't have it implemented in ROS, so what do you suggest instead?

Reply:
This is just the list of known interfaces. Why does it need to be implemented to be here?

Reply:
So I suppose your remark also applies for IID_IUserNotification2 (below).

Reply:
Yeah, hence "these" :)